website/static/js/citationGrid.js

```diff
@@ -219,10 +219,10 @@ var renderActions = function(item, col) {
             css: 'btn btn-default btn-xs',
             tooltip: 'Download citations',
             config: function(elm, isInit, ctx) {
-                var text = self.getCitations(item).join('\n');
+                var text = '{\\rtf1\\ansi\n' + self.getCitations(item, 'rtf').join('\\\n') + '\n}';
```

Comment:
Having spoken with you on this, I believe this is correct. If you could add some commentary as to why the string is being built the way it is, that would be helpful. Make sure to mention that backslashes are doubled because they're escaped in JS, and give a short example of properly-formatted RTF. (`\rtf1\ansi\n<content>\\n`)